README.md

```diff
@@ -35,17 +34,21 @@ $ docker run -ti -v `pwd`/fintud-macrep:/urbit/fintud-macrep paulbellamy/urbit f
 This can be useful for development
 
 ```
-$ docker run -ti paulbellamy/urbit -F -I zod -A /urbit/arvo -c mycomet
+$ docker run -ti paulbellamy/urbit -F -I zod -A /arvo -c mycomet
 ```
 
-There is another image, with a different entrpoint to make this even
-easier:
+There is another image, with a different entrypoint and the arvo repo checekd
+out to make this even easier:
 
-```
-$ docker run -ti paulbellamy/fakeurbit -c mycomet
-```
+    $ docker run -ti paulbellamy/urbit:fakezod -A /arvo -c piername
+
+to start it up after stopping it, omit `-A` and `-c`:
+
+    $ docker run -ti paulbellamy/urbit:fakezod piername
```

Comment:
I'm assuming the change from a separate image to a separate tag will mean re-setting up the dockerhub automated builds? It's not a problem, just checking.